src/pyproject_fmt/cli.py

```diff
@@ -88,6 +89,11 @@ def _build_cli() -> ArgumentParser:
         default=DEFAULT_INDENT,
         help="number of spaces to indent",
     )
+    parser.add_argument(
+        "--max-supported-python-is-prerelease",
```

Comment:
Can we instead just have the user specify the Mac supported Python explicitly here, please?

Reply:
Sure, so just make it an integer option?

Reply:
Yes 👍